stock_removal_location_by_priority/models/stock_quant.py

```diff
@@ -19,9 +19,9 @@ def _get_removal_strategy_order(self, removal_strategy):
             "stock_removal_location_by_priority.group_removal_priority"
         ):
             if removal_strategy == "fifo":
-                return "in_date ASC NULLS FIRST, removal_priority ASC, id"
```

Comment:
Why this change ?

Reply:
`in_date` field is required on 16.0 (it wasn't on 14.0) so I guessed it's no longer needed